### DISCUSSION TITLE
채팅봇관련 문의드립니다

### DISCUSSION THREAD

**Discussion by jngjsk** [Lastorder-DC/chatreader-kor#38] [문의] [answered]
안녕하세요 채팅봇 잘 사용하고있습니다. 한국어와 일본어로 스트리밍중입니다.
감사히 사용중에 염치없는 말씀입니다만 히라가나 없이 한자만 쓰면 중국어로 읽어서 사용에 불편을 겪고있습니다. 
예를 들면 ㅋㅋ의미의 풀 초(草) 한글자 채팅시 일본어 쿠사가 아닌 중국어 사오로 읽습니다.
혹시 읽는 언어의 우선순위를 바꾸거나, 다른 좋은 방법이 있을지 궁금해졌습니다.
좋은 하루 되시길 바랍니다.

**Comment by Lastorder-DC** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/18280396/136544520-8b9fbc75-4f96-4f84-846f-4c07bc61ffd0.png)
언어 선택 기능으로 구현되었습니다. 일본어를 우선해서 인식하고 싶으시다면 일본어 우선순위를 높게 잡아주시면 됩니다.
